### DISCUSSION TITLE
gives http 403

### DISCUSSION THREAD

**Discussion by mistysloth** [jeffalo/ocular#155] [Q&A]
![Screen Shot 2022-04-28 at 8 29 31 PM](https://user-images.githubusercontent.com/93560458/165868317-1ea7bddf-985a-46c1-b22d-8040565c0fc8.png)
Why?

**Comment by hello-smile6** [comment]
What page?

**Reply by mistysloth** [reply]
all pages

**Reply by hello-smile6** [reply]
Strange. Did your IP get blocked?

**Reply by mistysloth** [reply]
@jeffalo

**Reply by jdev082** [reply]
> _Censored IP Address_

DELETE THAT! NEVER SHARE YOUR IP ADDRESS!!!

**Reply by mistysloth** [reply]
oops

**Reply by mistysloth** [reply]
please also delete as it contains quote. @JaydenDev

**Reply by cobaltt7** [reply]
lmfao

**Comment by jeffalo** [comment]
Are you behind a school network or something? Perhaps ocular is blocked?

**Reply by mistysloth** [reply]
This is my home internet. I checked the firewall, blockers, and exstensions.

**Reply by hello-smile6** [reply]
@SecretAgent55 What DNS servers are you using?

**Reply by mistysloth** [reply]
8.8.4.4

**Reply by jeffalo** [reply]
That's really weird. I don't think that's anything on my end. Do you have trouble connecting to any other Cloudflare protected websites? How about https://my-ocular.jeffalo.net/api/user/jeffalo ?

**Comment by mistysloth** [comment]
all pages
________________________________
